### PR TITLE
Add Software Engineer Junior job mission based on Analytics changes

### PR DESCRIPTION
**Pull request by ryukinix**
Postgres it's deprecated. We don't want test this anymore, so this
it's just optional (and free for choice). The most important things
are:

+ API REST Development (input, ouput and interface)
+ Code simplicity (no code smell!)
+ Software Development with portability in mind (like Docker)
+ Tests

### CODE REVIEW

File: analytics/software-engineer-junior/challenge.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+{ "pk": "321", "score": "1" }
+{ "pk": "50", "score": "98" }
+```
+
+Para que o resultado do esforço dos cientistas de dados ganhe valor,
+esses dados devem ser injetados na plataforma para que outras
+aplicações façam uso desses indicadores. O meio de persistência é de
+sua escolha: apenas em memória *RAM* durante a execução, arquivo de
+texto ou um banco de dados como *SQLite* ou *PostgreSQL*. Mas note que
+aprioridade desse teste é a integração do pipeline proposto.
```
Comment (cadicallegari) [inline analytics/software-engineer-junior/challenge.md]: a missing white space  here =)

File: analytics/software-engineer-junior/challenge.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+A Missão
+--------
+
+* Desenvolver o serviço http que receberá esses dados e irá
+  persistí-los. O serviço também deverá permitir a consulta
```
Comment (cadicallegari) [inline analytics/software-engineer-junior/challenge.md:40]: maybe, move the explanation about how to do the persistence (using mem, file or db) to that session

File: analytics/software-engineer-junior/challenge.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  dos valores previamente guardados.
+
+* Desenvolver um cliente pra esse serviço que receba como entrada um
+  diretório contendo arquivos com os dados e envie para o serviço
+  http. Deverá conter um comando a parte também para consulta baseado
+  no atributo `pk`.
+
+Considerações
```
Comment (cadicallegari) [inline analytics/software-engineer-junior/challenge.md:53]: or even move the persistence choises to that session =P

File: analytics/software-engineer-junior/challenge.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+* [Docker](https://www.docker.com/) é largamente utilizado e
+gostaríamos muito de ver as suas habilidades com essa ferramenta.
+
+* Testar é legal! :) Gostamos de testes unitários.
+
+* Mostre-nos também tudo o que você sabe sobre boas práticas de
+  [git](https://git-scm.com).
+
+* Pense sempre na qualidade do código, tenha a manutenibilidade,
+legibilidade esimplicidade em mente. Lute contra
```
Comment (cadicallegari) [inline analytics/software-engineer-junior/challenge.md]: missing white space here

**Comment by ryukinix** [comment]
Ping @boechat107 @cadicallegari @lborguetti

**Comment by ryukinix** [comment]
Fixed @cadicallegari. It's done?

**Review by cadicallegari — APPROVED** [review]

**Comment by cadicallegari** [comment]
LGTM =)

**Pull request merged**